keyboards/aki27/cocot46plus/rules.mk

```diff
@@ -0,0 +1,7 @@
+SRC += matrix.c
+CUSTOM_MATRIX = lite
+POINTING_DEVICE_DRIVER = adns5050
+SPACE_CADET_ENABLE = no
+# A regular Pro Micro may work if this line is removed
+# but some features may need to be disabled for the firmware to fit in the storage.
+CONVERT_TO=promicro_rp2040
```

Comment:
Converters should not be enabled at the keyboard level.

(Any existing use in the repo has either already been removed, or is in the process of being removed.)

Reply:
Thanks for the feedback.

I tried removing this line and changing the `development_board` in `keyboard.json` from `promicro` to `promicro_rp2040`, but then I get compile errors. What's the proper way to get rid of `CONVERT_TO`?

```
Compiling: keyboards/aki27/cocot46plus/matrix.c                                                    In file included from <command-line>:
./keyboards/aki27/cocot46plus/config.h:30:27: error: 'D4' undeclared here (not in a function)
   30 | #define MATRIX_ROW_PINS { D4, C6, D7, E6, NO_PIN }
      |                           ^~
keyboards/aki27/cocot46plus/matrix.c:20:44: note: in expansion of macro 'MATRIX_ROW_PINS'
   20 | static const pin_t row_pins[MATRIX_ROWS] = MATRIX_ROW_PINS;
      |                                            ^~~~~~~~~~~~~~~
```

Reply:
You need to do all the other things that the converter does. In this case you will need to redo the pin assignments to match; RP2040 uses `GPx` notation.